torch/csrc/jit/passes/quantization/quantization_patterns.h

```diff
@@ -604,6 +604,18 @@ graph(%a_quant, %size, %align_corners, %scales_d, %scales_h, %scales_w):
           %r = aten::upsample_trilinear3d(%a_quant, %size, %align_corners, %scales_d, %scales_h, %scales_w)
           return (%r) )";
 
+  // aten::clamp
```

Comment:
For these types of ops, how are the scale and zero-point set in IR?

Reply:
we inherit the scale/zero_point from the previous quantized tensor, the code is in the first PR in the stack

Reply:
oh, actually, the code is already landed in the master

Reply:
https://codebrowser.bddppq.com/pytorch/pytorch/torch/csrc/jit/passes/quantization/insert_quant_dequant.cpp.html#643